bpf/enhancedrecording/command.bpf.c

```diff
@@ -165,7 +165,7 @@ static int exit_execve(int ret)
 }
 
 SEC("tp/syscalls/sys_execve")
-int tracepoint__syscalls__sys_enter_execve(struct trace_event_raw_sys_enter *tp)
+int tracepoint__syscalls__sys_enter_execve(struct syscall_trace_enter *tp)
```

Comment:
Could you add some background on what the fix is in the PR description?

This is quite mysterious to look at without any context.

Reply:
Added

Reply:
Thanks, that clears it up nicely!